server/routes/bookedtoday/summaryWithRecordController.ts

```diff
@@ -9,7 +9,10 @@ export default class SummaryWithRecordController {
       const { id } = req.params
       const arrival = await this.expectedArrivalsService.getArrival(id)
       const singleMatch = arrival.potentialMatches[0]
-      return res.render('pages/bookedtoday/summaryWithRecord.njk', { arrival, singleMatch })
+      const prisonerSummaryDetails = await this.expectedArrivalsService.getPrisonerSummaryDetails(
```

Comment:
Decided to keep 2 separate service calls for now as the page requires data from both the per and NOMIS.